ciongke/Makefile

```diff
@@ -0,0 +1,167 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# These are the usual GKE variables.
+PROJECT =
+ZONE = us-central1-f
+NUM_NODES = 3
+MACHINE_TYPE = n1-standard-8
+
+# These are GitHub credentials. Figure out TEAM_ID using the discover-team-id
+# script.
+HOOK_SECRET_FILE = $(shell echo $${HOME}/hook)
+OAUTH_SECRET_FILE = $(shell echo $${HOME}/k8s-oauth-token)
+ORG = kubernetes
+TEAM_NAME = kubernetes-maintainers
+TEAM_ID = 1674435
+
+# Don't forget to bump these versions when making a change.
+TEST_IMAGE_VERSION = 0.0
+HOOK_VERSION = 0.0
+TEST_PR_VERSION = 0.0
+
+RESULTS_BUCKET = $(PROJECT)-results
+SOURCE_BUCKET = $(PROJECT)-source
+
+# Useful rules:
+# - create-cluster turns up a cluster and the two GCS buckets then prints out
+#   the address to add to a GitHub webhook.
+# - delete-cluster deletes the cluster and the two GCS buckets.
+# - test-pr-image builds and pushes the latest test-pr image.
+# - hook-image builds and pushes the latest hook image.
+# - hook-deployment updates the deployment.
+# - hook-service create the hook service.
+# - test-image builds and pushes the test-image.
+
+create-cluster:
+	gcloud -q container --project "$(PROJECT)" clusters create ciongke --zone "$(ZONE)" --machine-type "$(MACHINE_TYPE)" --scope "https://www.googleapis.com/auth/compute","https://www.googleapis.com/auth/devstorage.full_control","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management" --num-nodes "$(NUM_NODES)" --network "default" --enable-cloud-logging --enable-cloud-monitoring
```

Comment:
Can we eliminate any/all of these scopes?
Also would it make sense to put it on its own network rather than default?